splunk-otel-android/src/test/java/com/splunk/rum/SlowRenderingDetectorImplTest.java

```diff
@@ -115,9 +126,14 @@ public void removeWithMetrics() {
 
         testInstance.add(activity);
 
+        ArgumentCaptor<SlowRenderingDetectorImpl.PerActivityListener> captor =
+                ArgumentCaptor.forClass(SlowRenderingDetectorImpl.PerActivityListener.class);
+
         for (long duration : makeSomeDurations()) {
             when(frameMetrics.getMetric(FrameMetrics.DRAW_DURATION)).thenReturn(duration);
-            testInstance.onFrameMetricsAvailable(null, frameMetrics, 0);
+            verify(activity.getWindow())
```

Comment:
This `verify` call could be made before the loop

Reply:
cool, probably reads a little cleaner that way.